README.md

```diff
@@ -1,150 +1,119 @@
+so that it can be included in the main web site.
 
-1.  Download the workshop website creation script from
-    [http://files.software-carpentry.org/swc-workshop-create](http://files.software-carpentry.org/swc-workshop-create).
+## Creating a Repository
 
-2.  Make sure that you are *not* inside another Git repository.
+1.  Go to [http://import.github.com](http://import.github.com).
 
-3.  Run the workshop website creation script with no parameters - it
-    will print a help message telling you what parameters it needs.
+2.  Enter the URL for this template repository, which is
+    `https://github.com/swcarpentry/workshop-template`.
 
-    You need to use
+3.  Check the URL.  (GitHub won't import until you've done this.)
 
-    ~~~
-    $ bash swc-workshop-create
-    ~~~
-
-    or change file mode with
-
-    ~~~
-    $ chmod +x swc-workshop-create
-    ~~~
-
-    before run the script by
-
-    ~~~
-    $ ./swc-workshop-create
-    ~~~
-
-4.  Run the script with those parameters.
-
-5.  Go into your newly-created repository.
-
-6.  Edit `index.html`.  Hints are embedded in the file, and full
-    instructions are in [CUSTOMIZATION.md](CUSTOMIZATION.md).
-
-7.  If you have installed the software described below:
-
-    1.  Check your changes by running `tools/check.py` inside your
-        repository.
-
-    2.  Preview your changes by running `tools/preview` and looking at
-        `_site/index.html`.
-
-    3.  For some links work properly, **including the one to Eventbrite**, you
-        will need to look at `_site/index.html` provide by a HTTP server that
-        can be done using
-
-        ~~~
-        $ jekyll server -d _site
-        ~~~
-
-        and look at http://localhost:4000.
+4.  Select the owner for your new repository.
+    (This will probably be you, but may instead be an organization you belong to.)
 
-8.  Commit your changes and push to the `gh-pages` branch of your
-    repository.
+5.  Choose a name for your workshop website repository.
+    This name should have the form `YYYY-MM-DD-site`,
+    e.g., `2015-07-01-miskatonic`.
 
-9.  Send the workshop coordinators the URL for your GitHub repository.
+6.  Make sure the repository is public.
 
-If the identifier for your workshop is `2015-07-01-esu`, and your
-GitHub username is `ghopper`, your workshop repository will be
-`https://github.com/ghopper/2015-07-01-esu` and the website for your
-workshop will be `https://ghopper.github.io/2015-07-01-esu`.  (Note:
-the workshop coordinators will want the former URL, not the latter.)
+7.  At this point, you should have a page like this:
 
-## Manual Setup
+    ![](http://software-carpentry.org/img/workshop-template/using-github-import.png)
 
-You can set up your repository manually instead of using the automated
-`create` script.  As above, we will assume that your user ID is
-`ghopper` and the identifier for your workshop is `2015-07-01-esu`.
+    You can now click "Begin Import".
```

Comment:
The import errors at 0% imported. I don't know if it is due to using 'http' instead of 'https' in the URL or if it is because I have a fork of workshop-template in my GitHub account. 

@wking Any thoughts? In the meantime, I will retry the 'https' link to see if the 500 error was a fluke. Overall, I like the new process :-)